src/lib/storage/disk_analyzer.rb

```diff
@@ -146,18 +141,25 @@ def find_installation_disks
       # @return [Array<string>] device names of candidate disks
       #
       def find_candidate_disks
-        dev_names(candidate_disks)
+        dev_names(candidate_disk_objects)
+      end
+
+      # Array with all disks in the devicegraph
+      #
+      # @return [Array<::Storage::Disk>]
+      def all_disks
+        # FIXME: to_a should not be necessary:
+        # libstorage should return something that Ruby can handle.
+        # This is very likely a problem of the Swig bindings.
```

Comment:
I just wanted to keep the comment added by @shundhammer, since it contains a `FIXME`. For me it's fine in the current state.